rosinstalls/indigo/gopher_desktop-stable.rosinstall

```diff
@@ -1,9 +1,23 @@
 # == Overview ==
 #
-# Software for testing, simulation, bootstrapping robot/simulation/concert/desktop software, no direct hardware dependencies here!
+# Software for simulations and remote/visual interactions with gopher robots. We all share in the development and maintenance
+# these packages.
 #
-# Bundling everything in gopher_desktop for now, but this could be split up.
-- git: { local-name: 'gopher_desktop', version: 'release/0.5-indigo',        uri: 'https://bitbucket.org/yujinrobot/gopher_desktop.git'}
-- git: { local-name: 'rocon_qt_gui',   version: 'release/0.9-indigo-gopher', uri: 'https://github.com/robotics-in-concert/rocon_qt_gui.git'}
-- git: { local-name: 'rqt_wrapper',    version: 'release/0.1-indigo',        uri: 'https://github.com/stonier/rqt_wrapper.git'}
-- git: { local-name: 'rqt_py_trees',   version: 'release/0.1-indigo',        uri: 'https://github.com/stonier/rqt_py_trees.git'}
+#   Permissions: open to all gopher developers - it's all dev/admin tools and no rocket science here
+#
+
+# Gopher podium/stage/gazebo simulations and rviz, rqt plugins
+- git: { local-name: 'gopher_desktop', version: 'release/0.6-indigo', uri: 'https://bitbucket.org/yujinrobot/gopher_desktop.git'}
+
+# Has the rocon remocon
+- git: { local-name: 'rocon_qt_gui',   version: 'release/0.10-indigo-gopher', uri: 'https://github.com/robotics-in-concert/rocon_qt_gui.git'}
+
+# Enabling library that keeps rqt gui's open even if the robot goes up and down
+- git: { local-name: 'rqt_wrapper',    version: 'release/0.2-indigo', uri: 'https://github.com/stonier/rqt_wrapper.git'}
+
+# Visualisation for behaviour trees
+- git: { local-name: 'rqt_py_trees',   version: 'release/0.2-indigo', uri: 'https://github.com/stonier/rqt_py_trees.git'}
+
+# A really nice plotting tool!
+# Awaiting https://github.com/ethz-asl/rqt_multiplot_plugin/pull/10
+#- git: { local-name: 'rqt_multiplot_plugin',   version: 'release/0.1-indigo', uri: 'https://github.com/yujinrobot/rqt_multiplot_plugin.git'}
```

Comment:
This [PR](http://repositories.ros.org/status_page/ros_indigo_default.html?q=rqt_multiplot) is merged and [deb](http://repositories.ros.org/status_page/ros_indigo_default.html?q=rqt_multiplot)is released. so, I comment this. Is it correct?